FaunaDB.Client.Test/EncoderTest.cs

```diff
@@ -397,5 +397,31 @@ public void TestStringOverride()
                 Encode(new StringOverride(testUri, testGuid))
             );
         }
+
+        [Test]
+        public void TestToStringTypes()
+        {
+            Assert.AreEqual(
+                "SetRefV({match: RefV(id = \"spells_by_element\", collection = RefV(id = \"indexes\")), terms: StringV(water)})",
+                new SetRefV(new Dictionary<string, Value>() {
+                    { "terms", StringV.Of("water") },
+                    { "match", new RefV("spells_by_element", new RefV("indexes")) }
+                }).ToString()
+            );
+
+            Assert.AreEqual("BooleanV(True)", BooleanV.Of(true).ToString());
```

Comment:
my vote is to remove the class name from the string representation, @faunaee will this break your tests?

Reply:
It won't break them, but I will have to update the expected output accordingly. I'm having to do that a fair bit anyway, so it's no problem. The opaque result was the problem.

Reply:
update: 
- simplified output (unwrapped the values)
- added `[]` to the Array
- dropped the `V` suffix for the remaining types (`Byte` and `Null`)